packages/auth/src/authClients/azureActiveDirectory.ts

```diff
@@ -0,0 +1,29 @@
+import { UserAgentApplication as AzureActiveDirectory } from 'msal'
+
+export type { AzureActiveDirectory }
+import type { AuthClient } from './'
+
+export type AzureActiveDirectoryClient = AzureActiveDirectory
+export interface AzureActiveDirectoryUser {}
+
+export const azureActiveDirectory = (client: AzureActiveDirectoryClient): AuthClient => {
+  return {
+    type: 'azureActiveDirectory',
+    client,
+    restoreAuthState: async () => {},
+    login: async (options?) => {
+      await client.loginPopup(options)
+    },
+    logout: (options?) => client.logout(options),
+    signup: async (options?) => {
+      await client.loginPopup(options)
+    },
+    getToken: async () => {
+      return sessionStorage.getItem('msal.idtoken')
+    },
+    getUserMetadata: async () => {
+      const user = await client.getAccount()
+      return user || null
+    },
```

Comment:
```suggestion
    getUserMetadata: async () => (await client.getAccount()) || null,
```

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/a5c608ac87daf42b574db1a1bbbeeec4a6620056